.github/workflows/supabase-branch.yml

```
@@ -0,0 +1,180 @@
+        run: |
+          echo "branch_name=pr-${{ github.event.number }}" >> $GITHUB_OUTPUT
+      
+      # Create or update Supabase branch environment
+      - name: Create/Update Supabase branch environment
+        id: supabase_branch
+        working-directory: lpg-backend
+        run: |
+          # Get Supabase project ID from Doppler
+          PROJECT_ID=$(doppler run --project lpg --config dev -- echo $SUPABASE_PROJECT_ID)
+          echo "project_id=$PROJECT_ID" >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)